sway-fmt-v2/src/fmt.rs

```diff
@@ -273,4 +273,50 @@ pub const TEST1: u16 = 10;"#;
             Formatter::format(&mut formatter, Arc::from(sway_code_to_format), None).unwrap();
         assert!(correct_sway_code == formatted_sway_code)
     }
+    #[test]
+    fn test_storage_without_alignment() {
+        let sway_code_to_format = r#"contract;
+
+storage {
+ long_var_name: Type1,
+      var2: Type2,
+}
+"#;
+        let correct_sway_code = r#"contract;
+
+storage {
+    long_var_name: Type1,
+    var2: Type2,
+}"#;
+
+        let mut formatter = Formatter::default();
+        let formatted_sway_code =
+            Formatter::format(&mut formatter, Arc::from(sway_code_to_format), None).unwrap();
+        println!("{}", formatted_sway_code);
+        println!("{}", correct_sway_code);
+        assert!(correct_sway_code == formatted_sway_code)
+    }
+    #[test]
+    fn test_storage_with_alignment() {
+        let sway_code_to_format = r#"contract;
+
+storage {
+ long_var_name: Type1,
+      var2: Type2,
+}
+"#;
+        let correct_sway_code = r#"contract;
+
```

Comment:
Could we get some tests that are on one line? e.g.

```rust
storage { foo: Test, bar: Test, baz: u64 } 
```

and maybe something more gross?
```rust
storage{foo:Test,bar
: 
    Test
, baz: u64 } 
```